task-maker-dag/src/dag.rs

```diff
@@ -75,6 +75,8 @@ pub struct ExecutionDAGData {
     pub execution_groups: HashMap<ExecutionGroupUuid, ExecutionGroup>,
     /// The configuration of this DAG.
     pub config: ExecutionDAGConfig,
+    /// Priority of this DAG.
+    pub priority: Priority,
```

Comment:
Nit: I'd like if you create a new type alias for this (`put type DagPriority = Priority`). It will make more clear the type of the items in the binary heap of the scheduler (see later).